fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -570,6 +570,10 @@ public String logKey() {
         ONLINE_INDEX_BUILDER_RECORDS_SCANNED("number of records scanned by online index build", false),
         /** The number of records indexed by {@link OnlineIndexer}. */
         ONLINE_INDEX_BUILDER_RECORDS_INDEXED("number of records indexed by online index build", false),
+        /** The number of indexer range termination by scan limit {@link OnlineIndexer}. */
+        ONLINE_INDEX_BUILDER_RANGE_BY_COUNT("number of indexer iterations terminated by scan limit", false),
+        /** The number of indexer range termination by write limit {@link OnlineIndexer}. */
```

Comment:
In a similar vein (presented mostly for parallelism, but this really should be whatever is decided for the first timer option, but adapted for this one).
```suggestion
        /** The number of {@OnlineIndexer} range scans terminated after hitting the by write limit. */
```